### ISSUE TITLE
Precision Attack dice roll

### ISSUE THREAD

**Issue by YenBenGrey** [jbowensii/More-Automated-Spells-Items-and-Feats#7]
With Precision Attack the description states that you can make a decision before or after the roll of the D20.

The window that pops up does not tell you the result of the D20, the player needs to keep an eye on the dice.

It isn't a major problem however just thought I would mention it. Unless there is something in maybe Midi I can tweak so they can see the number after the roll? In the pop up window.

**Comment by BalrogTaco** [comment]
I have a modified version that shows the roll.  But it also has a few other changes, so it may or may not work for you.  Change the attached file to .json to import it if you would like to try it out
[fvtt-Item-maneuvers_-precision-attack.txt](https://github.com/jbowensii/More-Automated-Spells-Items-and-Feats/files/10833952/fvtt-Item-maneuvers_-precision-attack.txt)

**Comment by YenBenGrey** [comment]
What I’ve done for now is set the time it takes for Dice so Nice to hide the dice for that one player to about 6 seconds. So they shouldn’t miss it. However I will give this a go. Thank you.

**Comment by YenBenGrey** [comment]
So far the above is amazing. 

Is it supposed to not pop up if the attack hits, because that is brilliant. And it says if the attack will MISS. One of the concerns from the player is that it pops up when not needed.

**Comment by jbowensii** [comment]
thank you [BalrogTaco] I will incorporate your changes this week and give you credit. :)

**Comment by BalrogTaco** [comment]
@YenBenGrey yes it is supposed to not show the dialog if you hit or if you are out of superiority dice (as tracked as a resource, not as another feat as some importers set up the resource)

@jbowensii hope it helps.  I think I have a few others of yours that I worked on as well, I suppose i should have made a PR

**Comment by jbowensii** [comment]
please send them over I will adjust and list you in the credits on the code and for the module..
